lib/matplotlib/legend.py

```diff
@@ -180,6 +180,8 @@ def __init__(self, parent, handles, labels,
                  title=None,  # set a title for the legend
 
                  framealpha=None,  # set frame alpha
+                 edgecolor=None,  # frame patch edgecolor
```

Comment:
No, they cannot.  `Axes.legend` has no mechanism for doing so.  Furthermore, even if there were such a mechanism, it would cause endless confusion because the order in which the kwargs are documented did not correspond to their order in the signature even before my PR.

The real bug in the present PR is that I added the new kwargs to the class docstring but not to the Axes method docstring.  I will rectify that now.